src/lib/drupal/lovell/staticProps.ts

```diff
@@ -159,7 +159,10 @@ async function getLovellListingPageStaticPropsResource(
         link: getLovellVariantOfUrl(item.link, context.lovell.variant),
       }
     }),
-  ]
+  ].sort((a, b) => {
+    return new Date(b.releaseDate).getTime() - new Date(a.releaseDate).getTime()
+  })
```

Comment:
Yeah, I don't...I don't understand why we need to add the sort to the query _and_ here. 😕 Does the query just not sort Lovell stories? Is the order maybe getting rejiggered somewhere along the way?